src/librustdoc/html/static/js/search.js

```diff
@@ -1546,7 +1720,9 @@ window.initSearch = function(rawSearchIndex) {
             if (searchState.input.value === "" && qSearch) {
                 searchState.input.value = qSearch;
             }
-            search();
+            if (hasClass(window.searchState.outputElement(), "hidden")) {
```

Comment:
Is this related to https://github.com/rust-lang/rust/issues/90454 ?

Reply:
It's a bad rebase, good catch!